Workbooks/MySQL Flexible Server/Auditing/Auditing.workbook

```diff
@@ -0,0 +1,225 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "**Administraive Actions on the service** (You need to send Azure Activity logs to Diagonistics Ref: https://docs.microsoft.com/en-us/azure/azure-monitor/essentials/diagnostic-settings?WT.mc_id=Portal-Microsoft_Azure_Monitoring&tabs=CMD)",
```

Comment:
nit: since it is a url, you can use the markdown url syntax to make this clickable

Reply:
Yes updated with FWLinks.